admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/controller/DeviceControllerTest.java

```diff
@@ -255,7 +255,7 @@ public void t004updateDeviceFailTest1() throws Exception {
 	public void t005getDevicesByRegistrationCenterTest() throws Exception {
 		
 		MasterDataTest.checkResponse(
-				mockMvc.perform(MockMvcRequestBuilders.get("/devices/mappeddevices/10001")).andReturn(), "KER-MSD-441");
```

Comment:
why for failed test cases we are adding null or modify the test name

Reply:
Modified the name sir.